components/create_post/create_post.jsx

```diff
@@ -275,8 +276,14 @@ class CreatePost extends React.PureComponent {
             /**
              * Function to set or unset emoji picker for last message
              */
-            emitShortcutReactToLastPostFrom: PropTypes.func
+            emitShortcutReactToLastPostFrom: PropTypes.func,
+
+            selectChannelMemberCountsByGroup: PropTypes.func,
         }).isRequired,
+
+        groupsWithAllowReference: PropTypes.object,
+        channelMemberCountsByGroup: PropTypes.object,
```

Comment:
same here might make sense to just make both of these default `{}`

Reply:
For `getChannelMemberCountsByGroup`, the selector already returns an empty `{}`.
For `groupsWithAllowReference`, I do a 
```
new Map(getAssociatedGroupsForReference(state, channel.team_id, channel.id).map((group) => [`@${group.name}`, group]))
```
Which if the array inside not present, it would get `new Map([])` which simply returns a empty Map without any entries.